core/drivers/scmi-msg/common.h

```diff
@@ -115,6 +115,15 @@ scmi_msg_handler_t scmi_msg_get_clock_handler(struct scmi_msg *msg);
 scmi_msg_handler_t scmi_msg_get_rd_handler(struct scmi_msg *msg);
 #endif
 
+#ifdef CFG_SCMI_MSG_VOLTAGE_DOMAIN
+/*
+ * scmi_msg_get_voltd_handler - Return a handler for a voltage domain message
+ * @msg - message to process
+ * Return a function handler for the message or NULL
+ */
+scmi_msg_handler_t scmi_msg_get_voltd_handler(struct scmi_msg *msg);
+#endif
```

Comment:
#else stub function?

Reply:
indeed!

Reply:
Actually stubs are not needed since call the discarded in ***scmi-msg/entry.c** use of `IS_ENABLED()`:
```c
	if (IS_ENABLED(CFG_SCMI_MSG_VOLTAGE_DOMAIN))
		scmi_msg_get_voltd_handler(msg);

Reply:
? Doesn't look safe/consistent.

Reply:
Stubs added in latest proposal and `if (IS_ENABLED())` removed.